### ISSUE TITLE
Fetch messages of a conversation

### ISSUE THREAD

**Issue by LeovR** [adiwajshing/Baileys#1606]
The functionality of fetching all messages of a conversation is not available anymore.
Could you provide a code example how this could be done?
I guess it could be solved via the generic `query` method but I don't know the protocol.

**Comment by catabozan** [comment]
The messages are stored in the `inMemoryStore` ([reference here](https://github.com/adiwajshing/Baileys#implementing-a-data-store)). All the messages are stored there when you connect with the qr code([you must implement it](https://github.com/adiwajshing/Baileys#implementing-a-data-store)) and when you receive/send new messages. 

You must query the messages from the store.

**Comment by catabozan** [comment]
Yes, when you connect with the QR code, all the messages(past messages) from all the (existing)conversations are fetched and stored in memory. Both groups and chats

**Comment by LeovR** [comment]
Ok. Thank you so far.
I will have a look at it again.
Maybe I missed the message updates.
I will close this for now.

**Comment by catabozan** [comment]
I think you are not getting all the messages from the chat because this package simulates a whatsapp web connection and the whatsapp web does not load all the messages. If you scroll enough in a chat you can see this 
![image](https://user-images.githubusercontent.com/43291246/169808787-35286d1b-6b70-4b3d-8327-066012d4d92c.png)

> Furthermore, is there an event which tells me I have now received all contacts, conversations and messages from WA?
I don't think there is a way to do that. This is a list of events supported by this package: [Event Handling](https://adiwajshing.github.io/Baileys/#handling-events)

**Comment by LeovR** [comment]
So currently there is no way to get all (including old) messages of a conversation as it was in the 3.x version?
And I think the legacy version isn't working anymore as well, is it?
